OCAT/client/pages/Assessments/NewAssessment.jsx

```diff
@@ -8,17 +8,37 @@ export const NewAssessment = () => {
   // create a form that utilizes the "onSubmit" function to send data to OCAT/client/services/AssessmentService.js and
   // then onto the OCAT/server/routes/AssessmentAPI express API
 
+  let score = 0;
```

Comment:
You shouldn't be storing this info in a let. You should be passing all your form data into the onSubmit function. You can calculate your score and risk level either in there or farther down in the submit process.